File: Core/CMS.Domain/Entities/MasterApostille.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CMS.Domain.Entities
{
    public class MasterApostille
    {
        [Key]
        public int ValueId { get; set; }
        public string ApostilleName { get; set; }
        public bool Status { get; set; }
        public bool IsDeleted { get; set; } = false;
    }
}

File: Frontend/CMS/src/app/models/master-apostille.ts
export class MasterApostille {
    valueId:number;
    apostilleName:string;
    status:boolean;
    isDeleted:boolean;

constructor(
    valueId:number=0,
    apostilleName:string='',
    status:boolean=true,
    isDeleted:boolean=false
){
    this.valueId=valueId;
    this.apostilleName=apostilleName;
    this.status=status;
    this.isDeleted=isDeleted;
}
}

export class MasterApostilleDto{
data: MasterApostille[];
  totalCount: number;

  constructor(data: MasterApostille[], totalCount: number) {
    this.data = data;
    this.totalCount = totalCount;
}
}

export class AddApostilleDto{
    apostilleName?:string;
    status?:boolean;
}


export class EditApostilleDto{
    apostilleName?:string;
    status?:boolean;
}

